src/workflows-app/RunDetails.js

```diff
@@ -120,6 +122,8 @@ export const BaseRunDetails = (
             }
           }
           const { workflowName } = metadata;
+          setStdOut(Object.values(metadata.calls)[0][0].stdout);
```

Comment:
Do you think its a good idea to handle the situation in which the `calls` array is empty? Yolo-indexing into arrays without checking first always makes me nervous.

Reply:
Lingo approved: "yolo indexing"

Reply:
Yup, that is a good idea, I added handling for when `calls` is empty.